src/App.js

```diff
@@ -35,4 +35,4 @@ function App() {
     );
 }
 
-export default withAuthenticator(App);
```

Comment:
there was no reason to change this. Think before you do bud

Reply:
changed this so the authenticator doesnt force you to sign in right away. Did think before I did. Maybe ask why instead of assuming its wrong.